pallets/unified-accounts/src/benchmarking.rs

```diff
@@ -83,4 +83,64 @@ mod benchmarks {
             .into(),
         );
     }
+
+    #[benchmark]
+    fn uam_to_account_id() {
```

Comment:
I'd suggest to drop the _uam\__ from the name.
It's clear from the context to which _WeightInfo_ implementation these calls refer to.